src/QMCWaveFunctions/BsplineFactory/BsplineSet.h

```diff
@@ -27,100 +28,98 @@
 
 namespace qmcplusplus
 {
-/** BsplineSet<SplineAdoptor>, a SPOSet
- * @tparam SplineAdoptor implements evaluation functions that matched the storage requirements.
- *
- * Equivalent to EinsplineSetExtended<Storage>
- * Storage is now handled by SplineAdoptor class that is specialized for precision, storage etc.
- * @todo Make SplineAdoptor be a member not the base class. This is needed
- * to make MultiBsplineSet (TBD) which has multiple SplineAdoptors for distributed
- * cases.
+/** BsplineSet is the base class for SplineC2C, SplineC2R, SplineR2R.
+ * Its derived template classes manage the storage and evaluation at given precision.
+ * BsplineSet also implements a few fallback routines in case optimized implementation is not necessary in the derived class.
  */
-template<typename SplineAdoptor>
-struct BsplineSet : public SPOSet, public SplineAdoptor
+struct BsplineSet : public SPOSet
```

Comment:
This is a good chance to bring this in line with the coding standards:

> Do not use struct as a way to avoid controlling access to the class. Only in rare cases where a class is a fully public data structure struct is this appropriate. Ignore (or fix one) the many examples of this in QMCPACK.

Reply:
Will update. Need to making builders friend classes. I wanted to move more code from this class to builders. So far these changes are not taking higher enough priority.

Reply:
Done.